website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -105,17 +99,23 @@ copies your environment’s Boundary **Cluster URL**.
 the copied Cluster URL.
 
 ```shell-session
-$ export BOUNDARY_ADDR=<boundary-cluster-url>
-```
+ $ export BOUNDARY_ADDR=<boundary-cluster-url>
```

Comment:
oof, looks like the code block fences didn't get applied properly for some reason, probably because github's suggestion blocks use tics as well

this should show up like this:

```shell-session
 $ export BOUNDARY_ADDR=<boundary-cluster-url>
```

I'll add a direct commit

Reply:
I just tried to fix that @stellarsquall . I think it might be OK in the preview deployment that's building now.